### ISSUE TITLE
Table block: row/column controls don't do anything if a table cell isn't previously focused

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#2077]
More an interaction flaw than a bug. To my understanding, the TinyMCE table plugin needs a reference to a table cell in order to be able to add/delete rows and columns. That is, when clicking on a table cell or tabbing with the keyboard to a table cell, it then "remembers" the cell that was focused. This is necessary to have a sort of "reference" to determine where to add rows/columns or delete them. 

Thus, the commands in the Table drop-down menu work only if a table cell had focus before using the controls. However, it is possible to make the toolbar and drop-down appear even when no cell has focus. This depends on the integration of the table plugin in Gutenberg, since in TinyMCE the implementation of the table controls is a bit different.

For example, clicking anywhere in the area highligthed in the screenshot below or tabbing without actually focusing the table cells:

<img width="792" alt="screen shot 2017-07-28 at 18 56 53" src="https://user-images.githubusercontent.com/1682452/28728702-e6c3708c-73c9-11e7-8a3d-b455866749c5.png">

To reproduce:
- click on the highlighted area
- the toolbar appeares
- open the drop-down menu
- click on any command to add/delete rows or columns
- nothing happens

this is potentially confusing for users, and even more confusing for assistive technologies users.

Not sure about the best option to address this issue. Maybe it should be addressed upstream for example: when there's no reference to a cell, fallback to the first cell in the first row?

**Comment by EphoxJames** [comment]
I have a fix for this in https://github.com/WordPress/gutenberg/pull/2143 waiting on reviewers.
This is related to https://github.com/WordPress/gutenberg/issues/1751 .

**Comment by afercia** [comment]
As long as it works well also when using the keyboard, the fix is very welcome! 🙂

**Comment by EphoxJames** [comment]
We probably need a keyboard shortcut for the various options or one to select the menu to make it nicer with the keyboard - at the moment it only theoretically works. It is technically easy to add shortcuts but knowing what key combinations make sense as shortcuts is a hard problem. I am open to suggestions.

Still that should be fixed in another pull request. Feel free to open an issue and CC me.
